core/src/mindustry/ui/fragments/BlockInventoryFragment.java

```diff
@@ -116,12 +118,20 @@ private void rebuild(boolean actions){
 
                 updateTablePosition();
                 if(tile.block().hasItems){
+                    boolean dirty = false;
+                    float dt = Time.delta();
```

Comment:
This variable is only used once, so just inline it.

Reply:
Whoops, didn't see that I wasn't using dt twice any more (I was earlier but then changed it).